lib/CL/devices/hsa/pocl-hsa.c

```diff
@@ -118,6 +126,112 @@ pocl_hsa_get_agents(hsa_agent_t agent, void *data)
   return HSA_STATUS_SUCCESS;
 }
 
+//Get hsa-unsupported device features from hsa device list.
+int num_hsa_device = 1;
+hsa_device_features hsa_device_lists[MAX_HSA_AGENTS];
+void init_hsa_device_list()
+{
+	hsa_device_lists[0].dev_name = "Spectre";
+	hsa_device_lists[0].llvm_cpu = "kaveri";
+	//FIXME: I am not sure if it is related to main memory.
+	hsa_device_lists[0].max_mem_alloc_size = 592969728;
+	hsa_device_lists[0].global_mem_size = 2371878912;
+	hsa_device_lists[0].vendor_id = 0x1002;
+	hsa_device_lists[0].global_mem_cache_type = 0x2;
+	hsa_device_lists[0].global_mem_cacheline_size = 64;
+	hsa_device_lists[0].max_compute_units = 8;
+	hsa_device_lists[0].max_clock_frequency = 720;
+	hsa_device_lists[0].max_conastant_buffer_size = 65536;
+	hsa_device_lists[0].local_mem_size = 32768;
+}
+
+void get_hsa_device_features(char* dev_name, struct _cl_device_id* dev)
+{
+	int i;
```

Comment:
Check style in this function; indentation and spacing after for/if and before/after = and <.